react/week4/src/components/MyRepos.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from "react";
+
+function MyRepos() {
+  const [repo, setRepo] = useState([]);
+  useEffect(() => {
+    const MY_URL = `https://api.github.com/users/MirSzabo/repos`;
+    (async () => {
+      try {
+        const myData = await fetch(`${MY_URL}`);
+        const myRepo = await myData.json();
+        setRepo(myRepo);
+      } catch (error) {
+        console.log(error);
+      }
+    })();
+  }, [setRepo]);
+
+  return (
+    <div>
+      <h2>My Repos</h2>
+      {repo.map(list => {

```

Comment:
The first param for the `map()` callback is the `currentValue` in that list Read more [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Parameters). It's confusing to call it a `list`. We can call it `item` instead, or if we map over a list called `repos`, the current item (singular) we call it `repo`.
```suggestion
      {repos.map(repo => {

```